src/test/java/carracing/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,51 @@
+package carracing.step2;
+
+import java.util.StringTokenizer;
+
+public class StringAddCalculator {
+
+    private static final char customDelimiterStarter = '/';
```

Comment:
```suggestion
    private static final char CUSTOM_DELIMITER_STARTER = '/';
```
상수는 일반적으로 대문자로 작성합니다 (복합어일 경우 언더바로 연결)
따로 안내받은 컨벤션이 있으시면 한번 확인해보시고, 아니면 [요 문서](https://myeonguni.tistory.com/1596)를 참고해보세요ㅎ